<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>193</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>521</width>
    <height>193</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>521</width>
    <height>193</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>AVR_COMPLIT 1.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #002238;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>521</width>
      <height>193</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #002238;
color: rgb(185, 185, 185);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Компилировать и Прошить</string>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>6</y>
        <width>143</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #002a3f;
color: #0462b1;</string>
      </property>
      <property name="text">
       <string>Программатор</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>154</x>
        <y>6</y>
        <width>143</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #002a3f;
color: #0462b1;</string>
      </property>
      <property name="text">
       <string>Микроконтроллер</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>301</x>
        <y>6</y>
        <width>209</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #002a3f;
color: #0462b1;</string>
      </property>
      <property name="text">
       <string>ЛОГИ</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>24</y>
        <width>145</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">combobox-popup: 0;
color: rgb(0, 122, 252);
background-color: rgb(0, 41, 67);
border:1px solid rgb(0, 41, 67);
border:1px solid #001c2b;</string>
      </property>
      <property name="maxVisibleItems">
       <number>5</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>153</x>
        <y>24</y>
        <width>145</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">combobox-popup: 0;
color: rgb(0, 122, 252);
background-color: rgb(0, 41, 67);
border:1px solid rgb(0, 41, 67);
border:1px solid #001c2b;</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="maxVisibleItems">
       <number>5</number>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>82</y>
        <width>254</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 122, 252);
background-color: rgb(0, 41, 67);
border:1px solid rgb(0, 41, 67);
border:1px solid #001c2b;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_5">
      <property name="geometry">
       <rect>
        <x>265</x>
        <y>82</y>
        <width>33</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-color: #08647e;*/
background-color: #002a3f;
border:1px solid #001c2b;
color: #017af6;</string>
      </property>
      <property name="text">
       <string>. . .</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>63</y>
        <width>289</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #002a3f;
color: #0462b1;</string>
      </property>
      <property name="text">
       <string>.C - Файл</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>24</y>
        <width>211</width>
        <height>129</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 122, 252);
color: rgb(0, 177, 252);
background-color: rgb(0, 41, 67);
border:1px solid rgb(0, 41, 67);
border:1px solid #001c2b;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_6">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>120</y>
        <width>87</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-color: #00afca;*/
color:#00afca;
border:1px solid #004f5a;

background-color:  #00444f; /* rgb(0, 116, 134);*/</string>
      </property>
      <property name="text">
       <string>OK</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Прошивание</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>63</y>
        <width>289</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #002a3f;
color: #0462b1;</string>
      </property>
      <property name="text">
       <string>.HEX - Файл</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>24</y>
        <width>145</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">combobox-popup: 0;
color: rgb(0, 122, 252);
background-color: rgb(0, 41, 67);
border:1px solid rgb(0, 41, 67);
border:1px solid #001c2b;</string>
      </property>
      <property name="maxVisibleItems">
       <number>5</number>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>153</x>
        <y>24</y>
        <width>145</width>
        <height>25</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">combobox-popup: 0;
color: rgb(0, 122, 252);
background-color: rgb(0, 41, 67);
border:1px solid rgb(0, 41, 67);
border:1px solid #001c2b;</string>
      </property>
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string/>
      </property>
      <property name="maxVisibleItems">
       <number>5</number>
      </property>
      <property name="duplicatesEnabled">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>8</x>
        <y>82</y>
        <width>254</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 122, 252);
background-color: rgb(0, 41, 67);
border:1px solid rgb(0, 41, 67);
border:1px solid #001c2b;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>265</x>
        <y>82</y>
        <width>33</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-color: #08647e;*/
background-color: #002a3f;
border:1px solid #001c2b;
color: #017af6;</string>
      </property>
      <property name="text">
       <string>. . .</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>24</y>
        <width>211</width>
        <height>129</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 122, 252);
color: rgb(0, 177, 252);
background-color: rgb(0, 41, 67);
border:1px solid rgb(0, 41, 67);
border:1px solid #001c2b;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>120</y>
        <width>87</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">/*background-color: #00afca;*/
color:#00afca;
border:1px solid #004f5a;

background-color:  #00444f; /* rgb(0, 116, 134);*/</string>
      </property>
      <property name="text">
       <string>Прошить</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>301</x>
        <y>6</y>
        <width>209</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #002a3f;
color: #0462b1;</string>
      </property>
      <property name="text">
       <string>ЛОГИ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>6</y>
        <width>143</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #002a3f;
color: #0462b1;</string>
      </property>
      <property name="text">
       <string>Программатор</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>154</x>
        <y>6</y>
        <width>143</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #002a3f;
color: #0462b1;</string>
      </property>
      <property name="text">
       <string>Микроконтроллер</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Компиляция</string>
     </attribute>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>12</x>
        <y>6</y>
        <width>281</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #002a3f;
color: #0462b1;</string>
      </property>
      <property name="text">
       <string>Микроконтроллер</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>246</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 122, 252);
background-color: rgb(0, 41, 67);
border:1px solid rgb(0, 41, 67);
border:1px solid #001c2b;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>91</x>
        <y>122</y>
        <width>111</width>
        <height>29</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">color:#00afca;
border:1px solid #004f5a;

background-color:  #00444f; </string>
      </property>
      <property name="text">
       <string>Компилировать</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>24</y>
        <width>284</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">combobox-popup: 0;
color: rgb(0, 122, 252);
background-color: rgb(0, 41, 67);
border:1px solid rgb(0, 41, 67);
border:1px solid #001c2b;</string>
      </property>
      <property name="maxVisibleItems">
       <number>5</number>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>80</y>
        <width>33</width>
        <height>21</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #002a3f;
border:1px solid #001c2b;
color: #017af6;</string>
      </property>
      <property name="text">
       <string>. . .</string>
      </property>
     </widget>
     <widget class="QTextBrowser" name="textBrowser_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>24</y>
        <width>211</width>
        <height>129</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 122, 252);
color: rgb(0, 177, 252);
background-color: rgb(0, 41, 67);
border:1px solid rgb(0, 41, 67);
border:1px solid #001c2b;</string>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Noto Sans'; font-size:8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>301</x>
        <y>6</y>
        <width>209</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #002a3f;
color: #0462b1;</string>
      </property>
      <property name="text">
       <string>ЛОГИ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>62</y>
        <width>281</width>
        <height>17</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #002a3f;
color: #0462b1;</string>
      </property>
      <property name="text">
       <string>.C - Файл</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
